packages/syria/README.md

```diff
@@ -0,0 +1,41 @@
+# SY Syria
+
+Contains all the governorates of Syria:
+
+- Aleppo
+- As-Suwayda
+- Darra
```

Comment:
Could you fix the typo (in the SVG file too) please? It's "Daraa" instead of "Darra"